examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -158,12 +163,14 @@ export class TaskList extends DataObject implements ITaskList {
         }
         const newTask = new Task(id, nameSharedString, prioritySharedCell);
         this.tasks.set(id, newTask);
+        this._fluidSync = true;
         this.emit("taskAdded", newTask);
     };
 
     private readonly handleTaskDeleted = (id: string): void => {
         const deletedTask = this.tasks.get(id);
         this.tasks.delete(id);
```

Comment:
Is this an async function that needs an await?

Reply:
No, it should be a generic flag that is toggled when fluid data receives data and save data.